src/liballoc/rc.rs

```diff
@@ -2007,7 +2006,7 @@ trait RcBoxPtr<T: ?Sized> {
         // The reference count will never be zero when this is called;
         // nevertheless, we insert an abort here to hint LLVM at
         // an otherwise missed optimization.
-        if strong == 0 || strong == usize::max_value() {
```

Comment:
Moving from max_value to MAX seems less justified than moving from the old module constants to the new associated constsants.

Reply:
Is there any reason to not change it? The docs for `max_value` states that it's soft deprecated. The fact that `usize::MAX` is shorter and at least as readable seems like a good reason to change.

Reply:
All those changes are in the same commit. Backing out only the `max_value` ones means I have to manually re-do all the `std::<type>` changes. Does not seem productive to spend time on working backwards towards something that is not recommended for new code.

Reply:
The PR does not change this code any longer.